console/src/main/java/org/georchestra/console/ws/utils/LogUtils.java

```diff
@@ -257,6 +257,10 @@ public void logOrgChanged(Org org, JSONObject json) throws IOException {
         if (!org.getNote().equals(json.optString(Org.JSON_NOTE))) {
             createAndLogDetails(id, Org.JSON_NOTE, org.getNote(), json.optString(Org.JSON_NOTE), type);
         }
+        // mail addres of org changed
+        if (!org.getMail().equals(json.optString(Org.JSON_MAIL))) {
```

Comment:
I haven't tested if this could lead to a NPE

Reply:
I suppose it won't as it is defined by default to empty string in `OrgExt.java`. Maybe it should be a good thing to switch variables in order to be sure to avoid it ? `optString` returns an empty string if not found so it should never lead to an NPE.